packages/webapp/src/components/Animals/AddAnimalsDetails/type.ts

```diff
@@ -55,9 +58,9 @@ export enum DetailsFields {
 }
 
 export type Option = {
-  [DetailsFields.TYPE]: ReactSelectOption<string>; // TODO: LF-4159
-  [DetailsFields.BREED]: ReactSelectOption<string>; // TODO: LF-4159
-  [DetailsFields.USE]: ReactSelectOption<number | string>; // TODO: LF-4159
+  [DetailsFields.TYPE]: AnimalSelectOption;
+  [DetailsFields.BREED]: AnimalSelectOption;
+  [DetailsFields.USE]: ReactSelectOption<number | string>;
```

Comment:
I'm not sure why I didn't add this, but the value is a number, isn't it? 😓
Stories will need to be updated as well, sorry 🙏

Reply:
Fixed, thank you!